src/cli/README.md

```diff
@@ -1166,6 +1167,17 @@ Set the customized data poll period for sleepy end device (milliseconds >= 10ms)
 Done
 ```
 
+### pskc [-p] \<key\>|\<passphrase\>
+
+with `-p` generate pskc from \<passphrase\>(UTF-8 encoded) together with current network name and extended PAN ID, otherwise set pskc as \<key\>(hex format). 
```

Comment:
```suggestion
With `-p` generate pskc from \<passphrase\> (UTF-8 encoded) together with current network name and extended PAN ID, otherwise set pskc as \<key\> (hex format). 
```